Exiled.API/Features/Round.cs

```diff
@@ -80,7 +80,7 @@ public static bool IsLobbyLocked
         public static int ChangedIntoZombies => RoundSummary.changed_into_zombies;
 
         /// <summary>
-        /// Gets the amount of rounds.
+        /// Gets the number of rounds since server startups.
```

Comment:
> Gets the number of rounds since the server started.
or
> Gets the number of completed rounds since the server started.
Depending on what it gets.